server/src/main/java/org/opensearch/index/translog/transfer/FileTransferTracker.java

```diff
@@ -26,22 +25,28 @@
 import java.util.stream.Collectors;
 
 /**
- * FileTransferTracker keeps track of translog files uploaded to the remote translog
+ * FileTransferTracker keeps track of generational translog files uploaded to the remote translog store
  */
 public class FileTransferTracker implements FileTransferListener {
 
     private final ConcurrentHashMap<String, TransferState> fileTransferTracker;
-    private final ShardId shardId;
+    private final ConcurrentHashMap<Long, TransferState> generationTransferTracker;
```

Comment:
These 2 maps can be referenced as just Map and have ConcurrentHashMap as implementation.